galaxy_ng/tests/integration/conftest.py

```diff
@@ -143,18 +143,18 @@ class AnsibleConfigFixture(dict):
     if is_stage_environment():
         PROFILES = {
             "basic_user": {
-                "username": {"vault_path": "secrets/qe/stage/users/aa_no_access",
+                "username": {"vault_path": "secrets/qe/stage/users/ansible-hub-qe-basic",
                              "vault_key": "username"},
-                "password": {"vault_path": "secrets/qe/stage/users/aa_no_access",
+                "password": {"vault_path": "secrets/qe/stage/users/ansible-hub-qe-basic",
                              "vault_key": "password"},
                 "token": None,
             },
             "partner_engineer": {
-                "username": {"vault_path": "secrets/qe/stage/users/ansible_insights",
+                "username": {"vault_path": "secrets/qe/stage/users/ansible-hub-qe",
                              "vault_key": "username"},
-                "password": {"vault_path": "secrets/qe/stage/users/ansible_insights",
+                "password": {"vault_path": "secrets/qe/stage/users/ansible-hub-qe",
                              "vault_key": "password"},
-                "token": {"vault_path": "secrets/qe/stage/users/ansible_insights",
+                "token": {"vault_path": "secrets/qe/stage/users/ansible-hub-qe",
                           "vault_key": "token"},
             },
             "org_admin": {
```

Comment:
can we also rename `secrets/qe/stage/users/rbac_org_admin` to the username used for the `org_admin` profile? I couldnt log in as `rbac_org_admin`

Reply:
done!! I've used the `ansible-hub-qe` user here, since that is also a org admin. let me know if that's fine or we need a new user with some other permissions

Reply:
It might be helpful to add these comments above each profile

```
# ns owner to autohubtest2, not in partner engineer group, not an SSO org admin
"basic_user"

# in partner engineer group, not an SSO org admin
"partner_engineer"

# an SSO org admin, not in partner engineer group
"org_admin"

# for stage env, this can be same user as partner_engineer profile
"admin"
```

- profile `basic_user` user ansible-hub-qe looks good
- profile `partner_engineer` user `ansible-hub-qe` should not be an org admin (sorry, I didn't catch this yesterday when I looked)
- profile `org_admin` user should be a separate user that is an org admin and not in partner engineering group. I tried logging in as `ansible-hub-qe-rbac` but the credentials did not work
- profile `admin` user `ansible-hub-qe` looks good